notebooks/photutils/03_aperture_photometry/03_aperture_photometry.ipynb

```diff
@@ -0,0 +1,970 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Unfortunately, you can't plot SkyApertures. However, you can use them just as easily to perform aperture photometry!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "----\n",
+    "## Performing Aperture Photometry"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that we have aperture objects that fit our data reasonably well, we can finally perform photometry with the `aperture_photometry` function. This function takes the following arguments:\n",
+    "\n",
+    "* **`data`** - the background-subtracted data array on which to perform photometry.\n",
+    "* **`apertures`** - an aperture object containing the aperture(s) to use for the photometry.\n",
+    "* **`error`** (optional) - an array of values that represent the pixel-wise Gaussian 1-sigma errors of the input data.\n",
+    "* **`mask`** (optional) - a mask for the `data` to exclude certain pixels from calculations.\n",
+    "* **`method`** (optional) - how to place the aperture(s) onto the pixel grid (see below).\n",
+    "* **`unit`** (optional) - unit of `data` and `error`.\n",
+    "* **`wcs`** (optional) - the WCS transformation to use if `apertures` is a `SkyAperture` object. \n",
+    "* **`subpixels`** (optional) - the factor by which pixels are resampled (see below).\n",
+    "\n",
+    "The following methods are the options for how to place apertures onto the data pixel grid:\n",
+    "\n",
+    "* **exact** (default) - calculate the exact fractional overlap of each aperture for each overlapping pixel. This method is the most accurate, but will also take the longest. \n",
+    "* **center** - a pixel is either entirely in or entirely out of the aperture, depending on whether the pixel center is inside or outside of the aperture.\n",
+    "* **subpixel** - a pixel is divided into `subpixels` x `subpixels` subpixels, each of which are considered to be entirely in or out of the aperture depending on whether its center is in or out of the aperture. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from photutils import aperture_photometry\n",
+    "from astropy.table import QTable"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's see what this looks like using the first aperture in our image:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The CCDData mask will be automatically applied\n",
+    "phot_datum = aperture_photometry(xdf_image, elliptical_apertures[0]) \n",
+    "phot_datum"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `aperture_sum` value is what reports the number of electron counts within the aperture: 3.47 e<sup>–</sup>/s.\n",
```

Comment:
The changes in the definition of `theta` above mean that you don't get this value; I get 0.9 e/s or so for this number now. I can only assume once the bug is fixed this'll go back to normal, but this is a good litmus test... Or the number needs fudging!